examples/step-61/step-61.cc

```diff
@@ -520,87 +586,69 @@ namespace Step61
 
   // @sect4{WGDarcyEquation<dim>::compute_pressure_error}
 
-  // This part is to calculate the $L_2$ error of the pressure.
+  // This part is to calculate the $L_2$ error of the pressure.  We
+  // define a vector that holds the norm of the error on each cell.
+  // Next, we use VectorTool::integrate_difference() to compute the
+  // error in the $L_2$ norm on each cell. However, we really only
+  // care about the error in the interior component of the solution
+  // vector (we can't even evaluate the interface pressures at the
+  // quadrature points because these are all located in the interiors
```

Comment:
```suggestion
  // quadrature points because these are all located in the interior
```
(I'm not entirely sure, though)